A3A/addons/core/functions/Revive/fn_handleDamage.sqf

```diff
@@ -122,13 +128,30 @@ else
 			_damage = 0.9;
 			if (_part in ["head","body"]) then
 			{
-				if !(_unit getVariable ["incapacitated",false]) then
+				if (_unit getVariable ["oneTimeRevive",false] isEqualTo false) then
```

Comment:
Regarding the check being present twice in fn_makeUnconscious, does this not mean that it could theoretically be triggered twice? Is there a reason for why we don't just have the oneTimeRevive check once at the top or something?